### PR TITLE
Update: Align button structure with other Adapt icon buttons (fixes #13)

### LINKED ISSUE #13: Align button with other Adapt icon buttons
As per #4, for consistency with other Adapt button icons I think we can lose `.scrollPrompt__btn-icon` so we just have the icon within the button. See below.
```
      <button className="btn-icon scrollPrompt__button" onClick={onScrollPromptClick}>
        <div className="icon" />
      </button>
```
Note, the CSS would also need updating to the following - [lines 9 - 11](https://github.com/swashbuck/adapt-scrollPrompt/blob/99777a3efe9f24c89e6f18a2190f6a8d0992152a/less/scrollPrompt.less#L9-L11).
```
  &__button .icon {
    .icon-controls-down;
  }
```

### PR DESCRIPTION
**Pull request by kirsty-hames**
Fixes #13 

Update button template for consistency with other Adapt button icons.

### CODE REVIEW

**Review by swashbuck — APPROVED** [review]
👍

File: less/scrollPrompt.less [hunk 1]
```diff
@@ -6,7 +6,7 @@
     align-items: center;
   }
 
-  &__btn-icon .icon {
+  &__button .icon {
```
Comment (StuartNicholls) [inline less/scrollPrompt.less]: button to btn? btn seems more commonly used.

File: templates/scrollPrompt.jsx [hunk 2]
```diff
@@ -10,9 +10,7 @@ export default function ScrollPrompt (props) {
   return (
     <div className="scrollPrompt__inner a11y-ignore" aria-hidden="true">
       <button className="btn-icon scrollPrompt__button" onClick={onScrollPromptClick}>
-        <div className="scrollPrompt__btn-icon">
-          <div className="icon" />
-        </div>
+        <div className="icon" />
```
Comment (StuartNicholls) [inline templates/scrollPrompt.jsx:13]: Does this need aria-hidden ?

File: less/scrollPrompt.less (hunk 1 above)
Comment (kirsty-hames) [reply less/scrollPrompt.less]: Agreed, I can't actually find any instances where we've used `__button` in Adapt. I'll amend this thanks.

File: templates/scrollPrompt.jsx (hunk 2 above)
Comment (kirsty-hames) [reply templates/scrollPrompt.jsx:13]: Normally yes but we have set this already on `.scrollPrompt__inner`.

**Pull request merged**